src/main/java/Entities/User.java

```diff
@@ -1,4 +1,20 @@
 package Entities;
 
-public interface User {
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class User {
+    private String Username;
+    private String password;
```

Comment:
Capitalization on Username
Kind of redundant but you /could/ make this one line? 

private String username, password;

Reply:
One liners can often lead to code which is a lot more difficult to debug; remember that primarily you write code to be read by other developers (maybe many months after you create the initial version).

The compiler doesn’t need one liners - in fact the good C and C++ compilers for instance will remove temporary variables and optimize the code far better than you can by writing one-liners.

Reply:
Thanks for your feedback Christina! I agree with Rafael, I think code in one line might be more difficult to read and understand later